docker_readme.txt

```diff
@@ -0,0 +1,9 @@
+Steps for Docker: 
+ 1. install docker or docker for windows 
+ 2. docker build . in the directory where docker file is located
+ 3. docker images to get image name (or set it when building)
+ 3. docker run -t -d -v /var/run/docker.sock:/var/run/docker.sock {Image Name}
+ 4. docker exec -it {containername} bash
+ 5. cd /home
+ 6. python spin-docker.py 
+ 7. Now you can exit the container and you'll see the apps containers created on host and running
```

Comment:
The application is not running on my laptop (OS-X). Need to investigate why.

Reply:
Resolved as of yesterday's session :)